feature-libs/product-configurator/rulebased/core/state/actions/configurator.action.ts

```diff
@@ -185,6 +185,12 @@ export class UpdateConfigurationFinalizeSuccess extends StateUtils.EntitySuccess
   }
 }
 
+/*
+ * UpdateConfigurationFinalFail contains `Fail` word in its name, but the purpose ot this action is not clear
+ * due to extending `EntitySuccessAction` instead of `EntitySuccessAction`
+ * That said, it's not know whether this action should implement `ErrorAction` or not.
+ * Follow-up ticket: CXSPA-7091
+ */
```

Comment:
1. Are you sure we want it to be an official JSDoc documentation of the class? (by using JSDoc - it looks like an intended, perhaps wasn't intended). Or it's just a developer's note?

2. explanations of eslint-disable, can be passed after `--` in the same line where the eslint-disable comment is

Reply:
It's a multi-line comment and it won't appear as JSDoc. An example: 
![image](https://github.com/user-attachments/assets/978b840b-8248-47c7-89f0-7fb28d93d532)

But I can do it the way you proposed.